chapter2/tests/t20/terraform/base/rds.tf

```diff
@@ -0,0 +1,31 @@
+module "wordpress_mysql_rds" {
+  source                              = "terraform-aws-modules/rds/aws"
+  version                             = "5.9.0"
+  identifier                          = local.labels.rds
+  engine                              = var.rds_engine
+  family                              = var.rds_family
+  major_engine_version                = var.rds_major_engine_version
+  engine_version                      = var.rds_engine_version
+  instance_class                      = var.rds_instance_class
+  allocated_storage                   = var.rds_allocated_storage
+  max_allocated_storage               = var.rds_max_allocated_storage
+  skip_final_snapshot                 = true
+  db_name                             = var.rds_db_name
+  username                            = var.rds_db_user
+  create_random_password              = false
+  password                            = random_password.passwords_list["db_password"].result
+  port                                = var.rds_port
+  publicly_accessible                 = false
```

Comment:
Same issue - should be parameterized, with default value being `false`

Reply:
> Same issue - should be parameterized, with default value being `false`

@dmitry-mightydevops 
Done